File: Entornos/Ariketak/Condicionales (Switch)/ariketa2.cpp
#include <stdio.h>
int main(){
    char eguna;

    printf("Sartu egunaren zenbakia(1-7): "); // Operadorea aukeratu.
    scanf("%c", &eguna);

    switch (eguna){
        case '1': // kasu bakoitzerako "programa" rxiki bat idazteko tokia
            printf("Astelehena");
            break;
        case '2':
            printf("Asteartea / Martitzena");
            break;
        case '3':
            printf("Asteazkena / Eguaztena");
            break;
        case '4':
            printf("Osteguna / Eguena");
            break;
        case '5':
            printf("Ostirala / Barikue");
            break;
        case '6':
            printf("Larunbata /Zapatua");
            break;
        case '7':
            printf("Igarndea / Domeka");
            break;
        default:
            /* If operator is other than +, -, * or /, error message is shown */
            printf("Error! Ez dago %c zenbakiarekin bat egiten duen egunik!", eguna);
            break;
    }
    return 0;
}

File: Entornos/Python/switch/ariketa2.py
def main_ariketa1():
    inputnum = int(input("Sartu zenbaki bat(1-7): "))
    if inputnum == 1:
        print("Astelehena")
    elif inputnum == 2:
        print("Asteartea")
    elif inputnum == 3:
        print("Asteazkena")
    elif inputnum ==4:
        print("Osteguna")
    elif inputnum == 5:
        print("Ostirala")
    elif inputnum == 6:
        print("Larunbata")
    elif inputnum == 7:
        print("Igandea")
    else:
        print("Sartutako zenbakiak ez du egun baliokiderik.")


if __name__ == "__main__":
    main_ariketa1()